diff -Naur fail2ban-fail2ban-a20d1f8/config/fail2ban.conf fail2ban-fail2ban-a20d1f8.tpg/config/fail2ban.conf
--- fail2ban-fail2ban-a20d1f8/config/fail2ban.conf	2011-11-29 03:46:04.000000000 +0000
+++ fail2ban-fail2ban-a20d1f8.tpg/config/fail2ban.conf	2012-01-15 18:39:35.000000000 +0000
@@ -22,7 +22,7 @@
 #          Only one log target can be specified.
 # Values:  STDOUT STDERR SYSLOG file  Default:  /var/log/fail2ban.log
 #
-logtarget = /var/log/fail2ban.log
+logtarget = SYSLOG
 
 # Option: socket
 # Notes.: Set the socket file. This is used to communicate with the daemon. Do
