diff -Naur fail2ban-fail2ban-a20d1f8/config/jail.conf fail2ban-fail2ban-a20d1f8.tpg/config/jail.conf
--- fail2ban-fail2ban-a20d1f8/config/jail.conf	2011-11-29 03:46:04.000000000 +0000
+++ fail2ban-fail2ban-a20d1f8.tpg/config/jail.conf	2012-01-15 18:33:20.000000000 +0000
@@ -42,12 +42,12 @@
 
 [ssh-iptables]
 
-enabled  = false
+enabled  = true
 filter   = sshd
 action   = iptables[name=SSH, port=ssh, protocol=tcp]
            sendmail-whois[name=SSH, dest=you@example.com, sender=fail2ban@example.com]
-logpath  = /var/log/sshd.log
-maxretry = 5
+logpath  = /var/log/auth.log
+maxretry = 3
 
 [proftpd-iptables]
 
