diff --git a/config/jail.conf b/config/jail.conf
index 9296b6a..7c71feb 100644
--- a/config/jail.conf
+++ b/config/jail.conf
@@ -110,7 +110,7 @@ logencoding = auto
 #
 # true:  jail will be enabled and log files will get monitored for changes
 # false: jail is not enabled
-enabled = false
+enabled = true
 
 
 # "filter" defines the filter to use by the jail.
@@ -185,7 +185,7 @@ action_cf_mwl = cloudflare[cfuser="%(cfemail)s", cftoken="%(cfapikey)s"]
                 %(mta)s-whois-lines[name=%(__name__)s, sender="%(sender)s", dest="%(destemail)s", logpath=%(logpath)s, chain="%(chain)s"]
 
 # Report block via blocklist.de fail2ban reporting service API
-# 
+#
 # See the IMPORTANT note in action.d/blocklist_de.conf for when to
 # use this action. Create a file jail.d/blocklist_de.local containing
 # [Init]
@@ -228,7 +228,8 @@ action = %(action_)s
 port    = ssh
 logpath = %(sshd_log)s
 backend = %(sshd_backend)s
-
+enabled = true
+maxretry = 3
 
 [sshd-ddos]
 # This jail corresponds to the standard configuration in Fail2ban.
@@ -332,7 +333,7 @@ logpath = /opt/openhab/logs/request.log
 port    = http,https
 logpath = %(nginx_error_log)s
 
-# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module` 
+# To use 'nginx-limit-req' jail you should have `ngx_http_limit_req_module`
 # and define `limit_req` and `limit_req_zone` as described in nginx documentation
 # http://nginx.org/en/docs/http/ngx_http_limit_req_module.html
 # or for example see in 'config/filter.d/nginx-limit-req.conf'
